formats/bear-bear2bk.ts

```diff
@@ -0,0 +1,80 @@
+import { FormatImporter } from "../format-importer";
+import { parseFilePath } from 'filesystem';
+import { BlobWriter, TextWriter } from "@zip.js/zip.js";
+import { Notice, normalizePath } from "obsidian";
+import { ImportResult } from '../main';
+
+const EXPORTED_ASSETS_FOLDER_NAME = 'bear-assets';
+
+export class Bear2bkImporter extends FormatImporter {
+  init() {
+    this.addFileChooserSetting('Bear2bk (.bear2bk)', ['bear2bk']);
+    this.addOutputLocationSetting('Bear');
+  }
+
+  async import(): Promise<void> {
+    let { files } = this;
+    if (files.length === 0) {
+      new Notice('Please pick at least one file to import.');
+      return;
+    }
+
+    let folder = await this.getOutputFolder();
+    if (!folder) {
+      new Notice('Please select a location to export to.');
+      return;
+    }
+
+    let results: ImportResult = {
+      total: 0,
+      skipped: [],
+      failed: []
+    };
+
+    // @ts-ignore
+    const attachmentsFolderPath = await this.createFolders(EXPORTED_ASSETS_FOLDER_NAME);
```

Comment:
I think we should just call this "assets", as a subfolder of the user-chosen output folder. Also, what's the need for ts-ignore for?